src/components/InputTodo.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-alert */
```

Comment:
(OPTIONAL) Consider removing these linter disabling lines from code. It will make your project much more professional.